website/app/controllers/show.js

```diff
@@ -8,18 +8,27 @@ import { inject as service } from '@ember/service';
 
 import { showdownConfig } from '../shared/showdown-config';
 
-const getTOCs = (container) => {
+const getAnchoredHeadings = (container) => {
```

Comment:
[a11y, blocker] links must have discernible text (accessible name) 

(FWIW, this was done incorrectly on the scrappy site too)